modules/index-service/src/test/java/org/opencastproject/index/service/resources/list/impl/ListProvidersScannerTest.java

```diff
@@ -162,36 +212,27 @@ public void testInstallInputOrgInPropertiesFileExpectsAddedToService() throws Ex
     EasyMock.expect(org2.getId()).andReturn("org2").anyTimes();
     EasyMock.replay(org2);
 
-    String listName = "BLACKLISTS.USERS.REASONS";
-    File file = new File(
-            ListProvidersScannerTest.class.getResource("/ListProvidersScannerTest-WithOrg.properties").toURI());
-    Capture<ResourceListProvider> resourceListProvider = new Capture<>();
-    Capture<String> captureListName = new Capture<>();
-    ListProvidersService listProvidersService = EasyMock.createNiceMock(ListProvidersService.class);
-    listProvidersService.addProvider(EasyMock.capture(captureListName), EasyMock.capture(resourceListProvider));
-    EasyMock.expectLastCall();
-    EasyMock.replay(listProvidersService);
+    EasyMock.expect(securityService.getOrganization()).andReturn(org1).anyTimes();
+    EasyMock.replay(securityService);
 
-    ListProvidersScanner listProvidersScanner = new ListProvidersScanner();
-    listProvidersScanner.setListProvidersService(listProvidersService);
+    File file = getResourceFile("/ListProvidersScannerTest-WithOrg.properties");
     listProvidersScanner.install(file);
 
     ResourceListQuery query = new ResourceListQueryImpl();
 
-    assertEquals(1, resourceListProvider.getValues().size());
-    assertEquals(listName, resourceListProvider.getValue().getListNames()[0]);
-    Map<String, String> stuff = resourceListProvider.getValue().getList(listName, query, org1);
-    for (String key : stuff.keySet()) {
-      logger.info("Key: {}, Value {}.", key, stuff.get(key));
+    assertEquals(1, listProvidersService.getAvailableProviders().size());
+    assertEquals(listName, listProvidersService.getAvailableProviders().get(0));
+    assertEquals("org1", org1.getId());
+    assertTrue("Provider is not registered", listProvidersService.hasProvider(listName, org1.getId()));
+    Map<String, String> dictionary = listProvidersService.getList(listName, query, false);
+    for (String key : dictionary.keySet()) {
+      logger.info("Key: {}, Value {}.", key, dictionary.get(key));
     }
```

Comment:
Can you remove this or make this conditional? it just slows down the tests otherwise and has no effect.

Reply:
Removed.